config/default/metrics_service.yaml

```diff
@@ -5,10 +5,10 @@ metadata:
   namespace: system
 spec:
   ports:
-  - name: https
+  - name: metrics
     port: 8443
```

Comment:
I think this can be changed to 8080 too

Reply:
You're right, seems to be more ideal as the targetPort is essentially 8080 too. Thanks @sutaakar